packages/mdc-icon-button/mdc-icon-button.scss

```diff
@@ -23,10 +23,4 @@
   @include mdc-states;
 }
 
-.mdc-icon-button--disabled {
```

Comment:
Should this be replaced with a `:disabled` selector instead of removed completely?

Reply:
The `:disabled` selector is in the mixins file. This was left over from `icon-toggle` and was used when the `icon-toggle` was on an element other than a `button` (`span`, `div`, etc). We dropped support for everything except for `button` and `a` tags, so this is no longer needed at all. Like `mdc-button` you can use `:disabled` for `button` elements to disable them. Anchor tags don't support disabled at all.

Reply:
Sweet, thanks for explaining!